native_client/deepspeech_compat.h

```diff
@@ -0,0 +1,97 @@
+#ifndef DEEPSPEECH_COMPAT_H
+#define DEEPSPEECH_COMPAT_H
```

Comment:
nit: mention the goal of this file? make it clear it's for deprecated codebases ?